frontend/packages/core/src/Table/table.tsx

```diff
@@ -206,8 +206,9 @@ const TableRow = ({
   cellDefault,
   responsive = false,
   colSpan,
+  ...props
 }: TableRowProps) => (
-  <StyledTableRow onClick={onClick} $responsive={responsive}>
+  <StyledTableRow onClick={onClick} $responsive={responsive} {...props}>
```

Comment:
Is this just so we can use `sx` or something else?

Reply:
It is to allow TableRow to be customised with styled and add a disabled style to the entire row, i.e. lower the opacity of the contents, my alternative was to wrap it with a styled component and target the MUI class but I figured this could be reusable.

Reply:
Makes sense